src/watchNPM.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request-promise')
+const _ = require('lodash')
+const ChangesStream = require('changes-stream')
+const clean = require('normalize-registry-metadata')
+const logger = require('~/src/logging').logger(module)
+
+// will hit up registry
+exports.checkWatcher = async function () { // TODO: fill the skeleton
+  console.log('todo')
+}
+
+// will publish to mq
+exports.handleChanges = async function ({channel, client, registry}) {
+  logger.info('saw change....todo handle logic')
+}
+
+exports.startNPMWatcher = async function ({channel, client, registry}) {
+  console.log(registry)
+  client.get('changes-stream', function (err, reply) {
```

Comment:
What exactly is stored at the `changes-stream` key? Also this is not going to return `changes` like you expect since `client.get` is async and you're not awaiting for it. It should be

```js
try {
    reply = await client.getAsync('changes-stream')
} catch (err) {
    // Handle the error here
    return
}
```